fdbserver/storageserver.actor.cpp

```diff
@@ -11220,7 +11220,7 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 
 					if (CLIENT_KNOBS->ENABLE_MUTATION_CHECKSUM && cloneReader.protocolVersion().hasMutationChecksum() &&
 					    msg.checksum.present()) {
-						const int32_t checksum = msg.calculateChecksum();
+						const uint32_t checksum = msg.calculateChecksum();
```

Comment:
If checksum mismatches, maybe we should throw an error to kill the storage server. The reason is that this mutation will not be applied and will cause inconsistency, which this SS itself can't fix.